### DISCUSSION TITLE
Issues for your own Provider

### DISCUSSION THREAD

**Discussion by 27pchrisl** [SocialiteProviders/Providers#863] [General]
Hi,

I'm the maintainer of the Saml2 provider, and sometimes people open issues about it but I don't know unless I happen to check the Issues list. Does anyone know a technique for monitoring the issues of a GitHub repo for keywords like "Saml2" ?

Thanks!

 - Chris

**Comment by 27pchrisl** [comment]
@atymic checking in case you have a nifty solution for this...

**Reply by atymic** [reply]
Unfortunately not aware sorry :(
